apps/site/assets/ts/projects/__tests__/RoutePillListTest.tsx

```diff
@@ -38,6 +38,16 @@ describe("routeToModeName", () => {
 
     expect(routeToModeName(greenLineBranchRoute)).toEqual("green-line");
   });
+
+  it("returns the correct name for all subway line indicator", () => {
+    expect(
+      routeToModeName({
+        mode: "subway",
+        id: "subway",
+        group: "line"
```

Comment:
Interesting. I would have expected subway to be classified in the `mode` group, not the `line` group. Is this what we're getting from the CMS or is this just something you wanted to test?

Reply:
![image](https://user-images.githubusercontent.com/688435/64557989-d8b21e00-d310-11e9-9fe4-ebad21223a96.png)

Reply:
the function doesn't actually use that field, I just got it from copy/paste from previous test

Reply:
I'm still a little confused on how a subway tag from the CMS could come out as "subway--line" when there is no metadata on the tag with "line" in the first place, but this work does resolve the issue to LGTM.

Reply:
there is a function that converts this data into class names and this cases was falling into a branch where `-line` was being added as a suffix